### DISCUSSION TITLE
Adding statistics to box plots

### DISCUSSION THREAD

**Discussion by AlbertoBernacchi** [ChiLiubio/microeco#419] [Q&A]
Hello Chi,
I'm currently trying your package because I find it much easier to navigate than other packages, so thank you very much for your work. However, I have encountered an issue: I am working with trans_abund class, trying to make a plot_box to see the difference between endophitic fungi collected from two different locations. I would like to add statistics to this graph, but if I try with ggpubr stat_compare_means, the values end up being crumpled up together in the middle of the graph and I can't figure out how to put them on top of the corresponding box. Hoping to have expressed myself, I will attach an image of the problem. The graph per se it's ok, but the statistics are not on top of each pair. Can you help me?
Thank you very much.
(This is my code: 
abundances_famiglie$plot_box(group = "Location", show_point=FALSE, xtext_angle = 30)
+stat_compare_means(label = "p.signif", tip.length = 0.05, symnum.args = list(cutpoints = c(0, 0.0001, 0.001, 0.01, 0.05, 1), symbols = c("p<0.0001", "p<0.001", "p<0.01", "p<0.05", "ns")), label.x.npc = "center", method = "wilcox.test")
+ scale_y_continuous(expand = expansion(mult = c(0, 0.1)))
![image](https://github.com/user-attachments/assets/c9169eb2-5b26-49f9-8608-be1752cb08e7)

**Comment by ChiLiubio** [comment]
Hi. Thanks. How about using trans_diff class to show the abundance and add the statistics? It is designed specifically for the combination of visualization and statistics.

**Reply by AlbertoBernacchi** [reply]
Hi, thank you very much for the quick response and the tip. I'm currently trying this way. Thank you again.
